releases.json

```diff
@@ -3952,4 +3953,4 @@
       "1.3.3-1"
     ]
   }
-}
+}
```

Comment:
what?

Reply:
my editor added a newline at the end of the line

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
ok mb  done

Reply:
Please squash changes into one logical change per commit -- don't make followup commits that back out of changes made in a previous commit.

You can do this with e.g. `git commit --amend && git push --force` to correct a mistake in the previous commit. See also `git rebase -i` to re-order and combine commits together via the "fixup" method.

Reply:
done!